pkg/util/kubernetes/apiserver/apiserver_nocompile.go

```diff
@@ -50,3 +50,9 @@ func GetMetadataMapBundleOnAllNodes(_ *APIClient) (map[string]interface{}, error
 	log.Errorf("GetMetadataMapBundleOnAllNodes not implemented %s", ErrNotCompiled.Error())
 	return nil, nil
 }
+
+// GetNodeLabels retrieves the labels of the queried node from the cache of the shared informer.
+func GetNodeLabels(nodeName string) (map[string]string, error) {
+	log.Errorf("GetNodeLabels not implemented %s", ErrNotCompiled.Error())
+	return nil, nil
```

Comment:
I guess `error` should not be `nil`?

Reply:
yeah, this is mostly for the !kubeapiserver build tag. I can clean that up in a diff PR as it would affect all the other methods.